pptx_generator/src/handlers/generate_pptx.py

```diff
@@ -113,6 +113,7 @@ def generate_pptx(event, context):
     destPath = pydash.get(json_data,'destPath')
```

Comment:
Follow python variable naming conventions for all these variable names. (ex: dataObj should be data_obj)